tests/operators/solr_api/delete_alias_test.py

```diff
@@ -61,31 +62,32 @@ def test_correct_init(self):
             solr_conn_id='solr_conn_id')
 
         self.assertEqual(task.names, ['fe', 'fi', 'fo'])
-        self.assertEqual(task.skip_from_last, 0)
+        self.assertEqual(task.skip_from_last, 1)
```

Comment:
@sensei100 I've gone ahead and also made the default behavior to be to always skip the last item (which I think will be a little counter intuitive in contexts outside our use-case, but also will be safer in our use-case).